### ISSUE TITLE
RO abuse

### ISSUE THREAD

**Issue by langri-sha** [JavaScriptRoom/culture#36]
I would kindly like to ask someone if they could intervene with an RO that keeps harassing me. 

`ssube` is basically constantly intimidating me with kicks and today I've been kicked for practically no reason whatsoever, to my opinion.

**Comment by Canop** [comment]
For reference,
- start of the discussion: http://chat.stackoverflow.com/transcript/message/31909996#31909996
- notable comment before kick: http://chat.stackoverflow.com/transcript/message/31910281#31910281

**Comment by kendfrey** [comment]
IMHO, the kick was fair.

**Comment by ssube** [comment]
I kicked you because you've made a habit of answering questions when you aren't familiar with the topic, often incorrectly and occasionally followed by a comment [stating as much](http://chat.stackoverflow.com/messages/31910281/history). This was not an isolated incident and we've asked you to stop before ([example 1](http://chat.stackoverflow.com/transcript/message/31039133#31039133), [example 2](http://chat.stackoverflow.com/transcript/17?m=31296081#31296081), [example 3](http://chat.stackoverflow.com/transcript/17?m=31396727#31396727), [example 4](http://chat.stackoverflow.com/transcript/17?m=30827892#30827892)).

When I [asked you to stop](http://chat.stackoverflow.com/transcript/message/31910307#31910307) you asked to be ignored, then [when I gave you a direct warning](http://chat.stackoverflow.com/transcript/message/31910469#31910469) you [considered it spam](http://chat.stackoverflow.com/transcript/message/31910485#31910485), so I kicked.

This is the inverse of being a help vampire, but equally problematic. Instead of asking for help without doing any work, you're providing "help" without any idea if it's correct (or sometimes without even knowing what they're asking about).

I'll leave it to the other ROs to decide if this was appropriate or not and what to do.

**Comment by langri-sha** [comment]
I wasn't born taught. Admittedly, I've once pushed a bad answer that was completely useless to a user months ago for cross-origin requests and haven't engaged in such conduct ever since. 

In today's episode, a user wants to load a large JSON array from response and I've merely suggested the use of streams from the spec as a potential approach to avoid loading the full data. I've been kicked because I merely asked a question: "what is Foo?", which, should it have been asked by anyone else, would have gone unnoticed by said RO.

From my perspective, it's people asking questions and people giving answers, having a nice chat. I'm here because I believe the RO's discretion to use their powers is disingenuous and I'll be happy to provide other instances where I have been harassed by the RO should someone request so, but I'm not inclined to pull out strands and bits until then.

**Comment by tentonaxe** [comment]
I don't see anything wrong with trying to help with something you aren't familiar with, as long as you aren't getting in the way of someone else who is familiar with it that is currently trying to help. From my point of view nothing was done that warranted a kick or even a warning.

**Comment by benjamingr** [comment]
I disagree with the kick. If you believe someone is acting stupid but not abusive - call them out on it.

ROs aren't supposed to use these tools for anything but abuse. I don't believe a kick should be used for that purpose.

_That said_ I generally stand by ssube's judgment in this case - he has proven himself very reasonable and I'm missing a lot of context.

**Comment by Canop** [comment]
In this case it's hard to tell whether the kick was fair. I won't try to decide it. I'd just like to suggest both involved users to be a little more cautious, either when helping if they're unsure of their specific knowledge, or in reacting to advises and kicking.

**Comment by ssube** [comment]
> If you believe someone is acting stupid but not abusive - call them out on it.
> ROs aren't supposed to use these tools for anything but abuse.

@benjamingr I asked him to stop twice, then kicked when he dismissed that warning as ["spamming."](http://chat.stackoverflow.com/transcript/message/31910485#31910485) IMO that reaction is abuse(ive enough to warrant a kick).

> as long as you aren't getting in the way of someone else who is familiar with it that is currently trying to help.

@tentonaxe which is (IMO) the situation. The advice is sufficiently off-base that there is a running joke about whether he's actually a spam bot. That's easily noise that's preventing new users from actually getting help.

**Comment by tentonaxe** [comment]
but a kick? that's absurd. Kicking someone for helping someone... If he's wrong tell him he's wrong.

**Comment by rlemon** [comment]
Remember, we don't kick people who link w3schools, we educate them. If
someone is giving bad/wrong advice don't be abrasive, steer them in the
right direction. "That is bad advice, and here is why" does a lot more good
than a kick.

**Comment by kendfrey** [comment]
The kick wasn't just because of bad advice. @langri-sha told @ssube to "please stop spamming me" which was pretty ill-advised, if not kick-worthy.

**Comment by MadaraUchiha** [comment]
I feel like there's a deeper history going on behind the scenes here. You don't say "just ignore me" or "stop spamming me" if it didn't happen several times already. 

Here's my take: Criticizing is fine, if you have something constructive to say. Telling people "you don't know anything" or "stop helping when you don't know" won't do anyone good. Educate.

On the flip side, receiving criticism is also important. If you were called on that you're jumping to conclusions to quickly and shooting wild solutions into the air, try to have that in mind in the future. 

I don't feel the kick was warranted, but then again, I don't feel the entire situation was warranted to begin with. I'm clearly missing most of the context here.

**Comment by langri-sha** [comment]
Example 1: I have no idea what's the issue here. I've said that due to what the user [said here](http://chat.stackoverflow.com/transcript/message/31039164#31039164), that he deleted his configuration file. More so, the RO obviously has no familiarity with how Babel supportive packages are organized, continues to suggest that [the entirety](http://chat.stackoverflow.com/transcript/message/31039157#31039157) of `babel-preset-es2015` is required for a Node.js 6 environment. I actually [saved the user](http://chat.stackoverflow.com/transcript/message/31039197#31039197) from their bad advice.

Example 2: I've intentionally recommended that the user uses loaders, since loaders are one of the principal design components of Webpack. More so, most template loaders follow a bespoke pattern to programmatically expose a callable to which a context can be passed for rendering (which is the basis of plugins such as `html-webpack-plugin` how they render user-supplied templates with a custom context). Not going further into the discussion, let's say Webpack users should always use loaders to reference assets, a statement everyone can agree with.

Example 3: [another RO asks if MongoDB has transactions](http://chat.stackoverflow.com/transcript/message/31396371#31396371) and I enter the discussion by saying that we're limited to [atomic operations only](http://chat.stackoverflow.com/transcript/message/31396411#31396411). Mind you, where in the context of databases and the RO is clearly referring to transaction support from relational databases. If they're talking about transactions in abstract, that would be redundant, since transactions in that form have been there since MongoDB's inception.

> A transaction is a sequence of operations performed as a single logical unit of work. A logical unit of work must exhibit four properties, called the atomicity, consistency, isolation, and durability (ACID) properties, to qualify as a transaction.

This obviously doesn't apply in MongoDB due to how operations are processed and fair read/write locks.  I've clearly outlined that, within the context of databases, there's an obvious dichotomy between transactions [in relational databases](http://chat.stackoverflow.com/transcript/message/31396465#31396465) and in MongoDB.

Furthermore, if you follow the discussion, the RO seems to suggest that MongoDB is schemaless, which I point out is incorrect. Stemming from the data model itself, a schema is tracked, at the DB level on the server, for each collection, for their indexes, where any modification blocks the entire server. 

Due to the fact that I knew that the RO, historically, bashed MongoDB due to his lack of understanding how this particular piece of technology is meant to be used, I came with best intentions to share what little I knew. Obviously, I came to a very late understanding that these two ROs were completely uninterested in having any advice shared with them. After their insults, I've kindly disclosed to them that I believe [that they were harsh](http://chat.stackoverflow.com/transcript/message/31396998#31396998) and I ceased any communication with them since June 28th, aside from the last 24 hours.

Example 4: May 28th, user asks how to [write and parse ES6](http://chat.stackoverflow.com/transcript/17?m=30827817#30827817). Webpack won't support it (unless you're on the beta distribution tag), it's AST doesn't support Babel's AST and neither do you need Babel for ES6. This is probably the origin from where the other RO, `luggage`, begins to abuse me as well.

#### TL;DR

So fine, I've been called shit, stupid, a bot, can't bother to remember what else or how many times, had both `ssube` and `luggage` team up together or harass me independently. I think I'm perfectly well aware of the room's rules and I cannot see how I've been breaking them. I've only received complaints from one other room participant and I learned from their critique. I've said that I've been treated harshly. I completely avoided having any discussions with them prior to the past 24 hours. 

I ignore them the past month, to the point of leaving the room when `ssube` is there. After I notice that the [user's stylesheet has errors](http://chat.stackoverflow.com/transcript/message/31787858#31787858) (message was sent to trash), I open my mouth to suggest that [they validate their sources](http://chat.stackoverflow.com/transcript/message/31788035#31788035) and [I get immediately attacked](http://chat.stackoverflow.com/transcript/message/31788056#31788056). I opened my mouth in their presence today, [I immediately get attacked again](http://chat.stackoverflow.com/transcript/message/31910307#31910307).

I've been given a direct warning to stop causing trouble to users by helping them. Fine, I stopped, I haven't provided any help since. Please stop berating two separate issues, mine following up with a personal request to the RO that they ignore me and discontinue their harassement, as being one and the same. It is incidental that it ensued after the warning, which I have clearly adhered to.

I am here because no other RO seems to condone other lawful evil ROs and trying to defend myself and resolve these issues with the RO and their group directly is obviously impossible. This obviously isn't going to stop.

Please help. Thanks.

**Comment by Cerbrus** [comment]
From what I read here, it look like Filip and ssube have a history of "colliding".

Regardless of if this specific kick was _warranted_ or not, I can speak from experience when I say that ssube shouldn't have kicked Filip.  
If you're involved in the argument, let someone else handle it.  
Someone impartial.

That said, this looks like a typical case of minor annoyances getting completely out of hand.
Both parties involved could've handled this better. In that, I agree with Madara: Stay nice, and try to _listen_ to each other.

**Comment by langri-sha** [comment]
I've pleaded with them, have not escalated any of their attacks, even engaged into ridiculous behavior such as feigning invisibility to avoid aggravating them with my presence. 

The help I'm expecting here is that someone either sensibly affirms the presented rules and the room's designation with them or that they be changed, providing the rest of us an opportunity to either agree or disagree with what they actually are.
